### PR TITLE
[WIP] Prepare Graphpocalypse for merge

### PR DESCRIPTION
**Pull request by sugakandrey**
Raising just to get CI running with emacs tests.

### CODE REVIEW

**Comment by fommil** [comment]
@sugakandrey you still want to keep this PR open?

**Comment by sugakandrey** [comment]
@fommil I'll merge latest 2.0 fixes and merge this I guess.

File: core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala [hunk 1]
```diff
@@ -17,10 +17,12 @@ import org.ensime.api._
 import org.ensime.fixture._
 import org.ensime.indexer._
 import org.ensime.indexer.SearchServiceTestUtils._
+import org.ensime.intg.ExcludeInTravis
 import org.ensime.util.EnsimeSpec
 import org.ensime.util.file._
 import org.ensime.vfs.EnsimeVFS
 
+@ExcludeInTravis
```
Comment (fommil) [inline core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala]: there is a more standard way to do this with scalatest... no need to use annotations

Comment (sugakandrey) [inline core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala]: afaik you have to use annotations to mark whole suit and not just a single test, but I may be wrong.

Comment (fommil) [inline core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala]: yes that's right, but that's ok. The slow tests tend to be pretty big, and we might be able to run just one of them, e.g. in reverse lookups.

File: testutil/src/main/scala/org/ensime/util/EnsimeSpec.scala [hunk 2]
```diff
@@ -29,12 +33,30 @@ trait EnsimeSpec extends FlatSpec
...
+  private val timer = new Timer(true)
+  timer.schedule(
+    new TimerTask {
+      override def run(): Unit = {
+        println(s"${self.getClass} is still running!")
+        Thread.getAllStackTraces.asScala.foreach {
+          case (thread, stacks) =>
+            println(thread.getName)
+            stacks.foreach { stack =>
+              println(s"\tat$stack")
```
Comment (fommil) [inline testutil/src/main/scala/org/ensime/util/EnsimeSpec.scala]: I introduced a silly typo here, should have  a space before `$stack`

File: testutil/src/main/scala/org/ensime/util/EnsimeSpec.scala [hunk 3]
```diff
@@ -49,7 +49,7 @@ trait EnsimeSpec extends FlatSpec
           case (thread, stacks) =>
             println(thread.getName)
             stacks.foreach { stack =>
-              println(s"\tat$stack")
+              println(s"\tat $stack")
```
Comment (fommil) [inline testutil/src/main/scala/org/ensime/util/EnsimeSpec.scala]: +1

**Pull request merged**